fala/templates/adviser/cookies.html

```diff
@@ -8,6 +8,25 @@
   <div class="govuk-grid-column-full">
     <div id="google_translate_element" class="govuk-!-margin-bottom-6"></div>
 
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
```

Comment:
https://design-system.service.gov.uk/components/notification-banner/ 

There is a notification banner you can use instead of just HTML